<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>818</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Barq Download Manager</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/images/icons/barq.png</normaloff>:/images/icons/barq.png</iconset>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QListWidget" name="lwCategories">
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="alternatingRowColors">
        <bool>false</bool>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QTableWidget" name="tblActive">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="showDropIndicator" stdset="0">
         <bool>false</bool>
        </property>
        <property name="dragDropOverwriteMode">
         <bool>false</bool>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="textElideMode">
         <enum>Qt::ElideMiddle</enum>
        </property>
        <property name="showGrid">
         <bool>false</bool>
        </property>
        <property name="gridStyle">
         <enum>Qt::NoPen</enum>
        </property>
        <property name="columnCount">
         <number>7</number>
        </property>
        <attribute name="horizontalHeaderVisible">
         <bool>true</bool>
        </attribute>
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>true</bool>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>100</number>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string>gid</string>
         </property>
         <property name="textAlignment">
          <set>AlignHCenter|AlignVCenter|AlignCenter</set>
         </property>
        </column>
        <column>
         <property name="text">
          <string>File Name</string>
         </property>
         <property name="textAlignment">
          <set>AlignHCenter|AlignVCenter|AlignCenter</set>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Status</string>
         </property>
         <property name="textAlignment">
          <set>AlignHCenter|AlignVCenter|AlignCenter</set>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Total Size</string>
         </property>
         <property name="textAlignment">
          <set>AlignHCenter|AlignVCenter|AlignCenter</set>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Progress</string>
         </property>
         <property name="textAlignment">
          <set>AlignHCenter|AlignVCenter|AlignCenter</set>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Speed</string>
         </property>
         <property name="textAlignment">
          <set>AlignHCenter|AlignVCenter|AlignCenter</set>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Time Left</string>
         </property>
         <property name="textAlignment">
          <set>AlignHCenter|AlignVCenter|AlignCenter</set>
         </property>
        </column>
       </widget>
       <widget class="QFrame" name="frame">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>150</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QTabWidget" name="btmTab">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="tab">
            <attribute name="title">
             <string>Uris</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QTableWidget" name="tblUris">
               <property name="alternatingRowColors">
                <bool>true</bool>
               </property>
               <property name="selectionBehavior">
                <enum>QAbstractItemView::SelectRows</enum>
               </property>
               <property name="gridStyle">
                <enum>Qt::DotLine</enum>
               </property>
               <column>
                <property name="text">
                 <string>Uris</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Status</string>
                </property>
               </column>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Global Download Limit</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cboxGlobalSpeed">
        <property name="currentIndex">
         <number>14</number>
        </property>
        <item>
         <property name="text">
          <string>5          Kb/Sec</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>10        Kb/Sec</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>15        Kb/Sec</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>20        Kb/Sec</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>30        Kb/Sec</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>40        Kb/Sec</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>50        Kb/Sec</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>60        Kb/Sec</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>70        Kb/Sec</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>80        Kb/Sec</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>90        Kb/Sec</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>100      Kb/Sec</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>150      Kb/Sec</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>200      Kb/Sec</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Unlimited</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>818</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNewDownload"/>
    <addaction name="actionRemoveDownload"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuDownload">
    <property name="title">
     <string>Download</string>
    </property>
    <addaction name="actionStart"/>
    <addaction name="actionStartAll"/>
    <addaction name="actionPause"/>
    <addaction name="actionPauseAll"/>
    <addaction name="separator"/>
    <addaction name="actionCleanList"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>&amp;Settings</string>
    </property>
    <addaction name="actionPreference"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout_aria2"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuDownload"/>
   <addaction name="menuSettings"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhHiddenText</set>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNewDownload"/>
   <addaction name="separator"/>
   <addaction name="actionStart"/>
   <addaction name="actionPause"/>
   <addaction name="actionStartAll"/>
   <addaction name="actionPauseAll"/>
   <addaction name="separator"/>
   <addaction name="actionCleanList"/>
   <addaction name="separator"/>
   <addaction name="actionPreference"/>
   <addaction name="separator"/>
   <addaction name="actionQuit"/>
  </widget>
  <action name="actionNewDownload">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/icons/project-development-new-template.png</normaloff>:/images/icons/project-development-new-template.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="toolTip">
    <string>Add New Download</string>
   </property>
  </action>
  <action name="actionRemoveDownload">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/icons/list-remove.png</normaloff>:/images/icons/list-remove.png</iconset>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/icons/media-playback-start.png</normaloff>:/images/icons/media-playback-start.png</iconset>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
   <property name="toolTip">
    <string>Start Selected Downlad</string>
   </property>
  </action>
  <action name="actionStartAll">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/icons/media-seek-forward.png</normaloff>:/images/icons/media-seek-forward.png</iconset>
   </property>
   <property name="text">
    <string>Start All</string>
   </property>
   <property name="toolTip">
    <string>Start All Downloads</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/icons/media-playback-pause.png</normaloff>:/images/icons/media-playback-pause.png</iconset>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
   <property name="toolTip">
    <string>Pause Selected Download</string>
   </property>
  </action>
  <action name="actionPauseAll">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/icons/media-record.png</normaloff>:/images/icons/media-record.png</iconset>
   </property>
   <property name="text">
    <string>Pause All</string>
   </property>
   <property name="toolTip">
    <string>Pause All Downloads</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/icons/application-exit.png</normaloff>:/images/icons/application-exit.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="toolTip">
    <string>Quit The Programe</string>
   </property>
  </action>
  <action name="actionPreference">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/icons/preferences-system.png</normaloff>:/images/icons/preferences-system.png</iconset>
   </property>
   <property name="text">
    <string>Preference</string>
   </property>
  </action>
  <action name="actionAbout_aria2">
   <property name="icon">
    <iconset>
     <normaloff>:/images/images/icons/barq.png</normaloff>:/images/images/icons/barq.png</iconset>
   </property>
   <property name="text">
    <string>About aria2</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionDownloadLimit">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/icons/go-down.png</normaloff>:/images/icons/go-down.png</iconset>
   </property>
   <property name="text">
    <string>Download Limit</string>
   </property>
  </action>
  <action name="actionCleanList">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/icons/edit-clear.png</normaloff>:/images/icons/edit-clear.png</iconset>
   </property>
   <property name="text">
    <string>Clean List</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;remove all downloads with status (complete , error , removed)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>addNewDownload()</slot>
  <slot>setGlobalSpeed()</slot>
  <slot>pauseAll()</slot>
  <slot>unpauseAll()</slot>
  <slot>getActiveDownloads()</slot>
  <slot>pauseGid()</slot>
  <slot>unpauseGid()</slot>
  <slot>showPreferenceDialog()</slot>
  <slot>removeDownload()</slot>
  <slot>showAboutQt()</slot>
  <slot>showAboutAria2()</slot>
 </slots>
</ui>
